src/components/search/search.jsx

```diff
@@ -0,0 +1,83 @@
+import { cn } from '@/utilities/functions';
+import { File, Folder, Search, Slash, SquareSlash, X } from 'lucide-react';
+import Input from '../input';
+import { useState } from 'react';
+import { Button, Label } from '..';
+
+const SearchBox = ({
+	onChange = () => { },
+	placeholder = "Search",
+	variant = 'primary', // primary, secondary, ghost
+	disabled = false,
+	size = "sm", // sm, md, lg
+	loading = false,
+	loadingIcon = {}, // Icon shown during loading
+	searchResult = ['searchResult 1', 'searchResult 2', 'searchResult 3'],
+	additionalResult = ['Category 1', 'Category 2', 'Category 3'],
+	searchResultIcon = < File size={16} strokeWidth={2} />,
+	additionalResultIcon = <Folder size={16} strokeWidth={2} />,
+	className
+}) => {
+	const [isOpen, setIsOpen] = useState(false);
+
+
+	return (
+		<div className={cn('relative flex flex-col focus-within:z-10', className)}>
+			<Input
+				prefix={<Search />}
```

Comment:
We should be using input type search here - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search